qtkeychain/keychain.cpp

```diff
@@ -35,6 +35,11 @@ void Job::setSettings(QSettings *settings)
 
 void Job::start()
 {
+    if (d->started) {
+        qCritical() << "Starting the same twice. This is not supported, aborting.";
```

Comment:
I'd give some more context, like "Starting the same QtKeychain job twice". Otherwise looks good